.github/workflows/report-stable.yml

```diff
@@ -15,8 +15,11 @@ jobs:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
       with:
         fetch-depth: 0
-    - name: Run makefile
-      run: make api-report-issuetemplate  RESULTS_DIR=_results
+    - id: gh-issue-template
+      run: |
+        readarray out < <(make -s api-report-issuetemplate  RESULTS_DIR=_results)
```

Comment:
Why not just write the title into a separate file?
This section still has more shell acrobatics than I like to read :-)

Reply:
I've modified the script to create separates files for title and body in the updated version.